developer_guide/installation/system_requirements/system_install_osx_1011.rst

```diff
@@ -0,0 +1,279 @@
+System install on OS X 10.11 (and higher)
+=========================================
+
+Here is a quick guide to setup the :doc:`system_requirements` on OS X 10.11. This guide will help you to install all
+the packages and modules needed for Akeneo PIM, then configure them to match your local installation.
+
+System installation
+-------------------
+
+MySQL installation
+******************
+
+Akeneo PIM supports MySQL from 5.1 to 5.7.
+You have two possibilities:
+
+* Work with MySQL 5.6. You need to downgrade your version to MySQL 5.6.
+* Work with MySQL 5.7. MySQL 5.7 is not officially supported but works in experimental mode if you disable the ONLY_FULL_GROUP_BY mode.
+
+Download and install the MySQL Community edition server from the official website: https://dev.MySQL.com/downloads/MySQL/
+
+Select your Mac OS X platform then download and install the DMG Archive. This package installs a preference pane extension for MySQL.
+
+.. note::
+
+    You will be given the root password during the installation process. Don't forget to copy it in order to change it once the installation is over.
+    The MySQL files will be installed at the following location `/usr/local/MySQL`.
+
+Run the MySQL server on your machine by starting it from the preferences.
+
+Check that you can connect to MySQL with:
+
+.. code-block:: bash
+
+        $ MySQL -u root -p # Enter the copied password
+
+.. note::
+
+    During the first connection, MySQL will ask you to change the root password for security reasons, follow the indicated steps.
+
+PHP installation
+****************
+
+Since Akeneo PIM 1.6, the minimal PHP version is PHP 5.6.
```

Comment:
Then, we should not talk about php 7.0 and 7.1. With this sentence, you can think that it will works with php 7.0/7.1 too. And that's not the case !